.jekyll-cache/Jekyll/Cache/Jekyll--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910

```diff
@@ -0,0 +1 @@
+I"+{"source"=>"/Users/jgarnett/java/geoserver/website", "destination"=>"/Users/jgarnett/java/geoserver/website/_site", "collections_dir"=>"", "cache_dir"=>".jekyll-cache", "plugins_dir"=>"_plugins", "layouts_dir"=>"_layouts", "data_dir"=>"_data", "includes_dir"=>"_includes", "collections"=>{"posts"=>{"output"=>true, "permalink"=>"/:categories/:year/:month/:day/:title:output_ext"}}, "safe"=>false, "include"=>[".htaccess"], "exclude"=>[".sass-cache", ".jekyll-cache", "gemfiles", "Gemfile", "Gemfile.lock", "node_modules", "vendor/bundle/", "vendor/cache/", "vendor/gems/", "vendor/ruby/"], "keep_files"=>[".git", ".svn"], "encoding"=>"utf-8", "markdown_ext"=>"markdown,mkdown,mkdn,mkd,md", "strict_front_matter"=>false, "show_drafts"=>nil, "limit_posts"=>0, "future"=>false, "unpublished"=>false, "whitelist"=>[], "plugins"=>[], "markdown"=>"kramdown", "highlighter"=>"rouge", "lsi"=>false, "excerpt_separator"=>"\n\n", "incremental"=>false, "detach"=>false, "port"=>"4000", "host"=>"127.0.0.1", "baseurl"=>nil, "show_dir_listing"=>false, "permalink"=>"date", "paginate_path"=>"/page:num", "timezone"=>nil, "quiet"=>false, "verbose"=>false, "defaults"=>[], "liquid"=>{"error_mode"=>"warn", "strict_filters"=>false, "strict_variables"=>false}, "kramdown"=>{"auto_ids"=>true, "toc_levels"=>[1, 2, 3, 4, 5, 6], "entity_output"=>"as_char", "smart_quotes"=>"lsquo,rsquo,ldquo,rdquo", "input"=>"GFM", "hard_wrap"=>false, "guess_lang"=>true, "footnote_nr"=>1, "show_warnings"=>false}, "blog_url"=>"http://blog.geoserver.org", "jira_url"=>"https://osgeo-org.atlassian.net/projects/GEOS", "wiki_url"=>"https://github.com/geoserver/geoserver/wiki", "git_url"=>"https://github.com/geoserver/geoserver", "sf_url"=>"http://sourceforge.net/projects/geoserver", "nightly_url"=>"https://build.geoserver.org/geoserver", "docs_url"=>"http://docs.geoserver.org", "stable_version"=>"2.17.1", "stable_branch"=>"2.17.x", "maintain_version"=>"2.16.3", "maintain_branch"=>"2.16.x", "dev_version"=>nil, "dev_branch"=>"master", "watch"=>true, "livereload_port"=>35729, "serving"=>true, "url"=>"http://localhost:4000"}:ET
```

Comment:
the `.jekyll-cache` directory should probably be added to the `.gitignore` file and not commited to source control